pkg/api/types.go

```diff
@@ -2669,6 +2673,69 @@ type SecurityContext struct {
 	ReadOnlyRootFilesystem *bool `json:"readOnlyRootFilesystem,omitempty"`
 }
 
+// SeccompProfile represents a seccomp profile. A profile is composed of a list of seccomp
+// rules and a default action. Seccomp profiles are unnamespaced.
+type SeccompProfile struct {
+	unversioned.TypeMeta `json:",inline"`
+	ObjectMeta           `json:"metadata,omitempty"`
+
+	// The default action that should be taken if there are no rules defined for a system call.
+	DefaultAction SeccompAction `json:"defaultAction"`
+	// The rules for this profile.
+	Rules []SeccompRule `json:"rules,omitempty"`
+}
+
+// SeccompRule is a seccomp syscall filter rule.
+type SeccompRule struct {
+	// The name of the system call to use.
+	Name string `json:"name"`
+	// The action to take.
+	Action SeccompAction `json:"action"`
+	// The arguments to the filter.
+	Args []*SeccompArg `json:"args"`
```

Comment:
omitempty?  Or do you want it printed in JSON when there's no value?  I.e. is it optional or required?